src/modules/data/entities/aggregate/common/metagame.territory.result.embed.ts

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable @typescript-eslint/explicit-member-accessibility,@typescript-eslint/naming-convention */
 import {Column} from 'typeorm';
+import {Faction} from '../../../constants/faction.consts';
 
 export default class MetagameTerritoryResultEmbed {
```

Comment:
Add swagger docs here (ApiProperty) for each column, It'll show up as the example so for the fields using the embed class.

Reply:
3b410f2269ebc20d461ee4022d7e81f715fc642b